pages/dkp/kommander/2.3/clusters/custom-domain-certificate/configuration/index.md

```diff
@@ -0,0 +1,123 @@
+
+To customize the domain or certificate of a cluster, alter the `Ingress` value of the `KommanderCluster`. Note that the `Issuer` object must be created on the cluster where you want to customize the configuration. In the Management cluster, both the `KommanderCluster` and `Issuer` objects are on the same cluster. In Managed and Attached clusters, the `KommanderCluster` object is stored on Management cluster, and the `Issuer` object is on the Managed or Attached cluster.
+
+Use the API yaml to customize the domain (via the `hostname` field), the certificate (via the `issuerRef` or `certificateSecretRef` field), or both.
+
+For this, refer to the following examples:
+
+1.  You have two options to create or update and apply the `KommanderCluster` object with the wanted ingress.
+
+    One option is to use a certificate that is managed automatically and supported by cert-manager like ACME (if you use Let's Encrypt, refer to the [example below](#example-configure-a-custom-certificate-with-lets-encrypt). For this, reference the `Issuer` or `ClusterIssuer` on the target cluster **to be used by cert-manager** in the `issuerRef` field, and enter the custom domain in the `hostname` field of the target cluster:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: kommander.mesosphere.io/v1beta1
+    kind: KommanderCluster
+    metadata:
+      name: <cluster_name>
+      namespace: <workspace_namespace>
+    spec:
+      kubeconfigRef:
+        name: <cluster_name>-kubeconfig
+      clusterRef:
+        capiCluster:
+          name: <cluster_name>
+    ingress:
+      hostname: <cluster_hostname>
+      issuerRef:
+        namespace: <issuer_namespace>
+        name: <issuer_name>
+    EOF
+    ```
+
+    Another option is to use a certificate provided by you and **customized for your hostname**. To do so, create a secret holding the certificate on the target cluster. Reference that secret in the `certificateSecretRef` field and the custom domain in the `hostname` field of the target cluster:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: kommander.mesosphere.io/v1beta1
+    kind: KommanderCluster
+    metadata:
+      name: <cluster_name>
+      namespace: <workspace_namespace>
+    spec:
+      kubeconfigRef:
+        name: <cluster_name>-kubeconfig
+      clusterRef:
+        capiCluster:
+          name: <cluster_name>
+    ingress:
+      hostname: <cluster_hostname>
+      certificateSecretRef:
+        namespace: <secret_namespace>
+        name: <secret_name>
+    EOF
+    ```
+
+### Example: Configure a custom certificate with Let's Encrypt
+
+Let's Encrypt is one of the Certificate Authorities (CA) supported by cert-manager. To set up a Let's Encrypt certificate, provide a `Issuer` or `ClusterIssuer` on the target cluster in the `issuerRef` field:
+
+<!-- TODO: example for let's encrypt -->
```

Comment:
@mhrabovcin @floridoo Could you provide the example command for let's encrypt?

Reply:
```
cat <<EOF | kubectl apply -f -
apiVersion: cert-manager.io/v1
kind: ClusterIssuer
metadata:
  name: custom-acme-issuer
spec:
  acme:
    email: <your_email>
    server: https://acme-v02.api.letsencrypt.org/directory
    privateKeySecretRef:
      name: kommander-acme-issuer-account
    solvers:
      - dns01:
          route53:
            region: us-east-1
            role: arn:aws:iam::YYYYYYYYYYYY:role/dns-manager
EOF
```

```
cat <<EOF | kubectl -n <workspace_namespace> --kubeconfig <management_cluster_kubeconfig> patch \ 
    kommandercluster <cluster_name>  --type='merge' --patch-file=/dev/stdin
spec:
  ingress:
    hostname: custom-domain.example.com
    issuerRef:
      name: custom-acme-issuer
      kind: ClusterIssuer
EOF
```